web/crux-ui/src/components/products/versions/images/edit-image-json.tsx

```diff
@@ -34,8 +34,12 @@ const EditImageJson = (props: EditImageJsonProps) => {
 
   const throttle = useThrottling(IMAGE_WS_REQUEST_DELAY)
 
+  const [jsonError, setJsonError] = useState<boolean>(false)
+
   const onMergeValues = (_: JsonConfig, local: JsonConfig): JsonConfig => {
-    onPatch(mergeJsonConfigToContainerConfig(config, local))
+    if (!jsonError) {
+      onPatch(mergeJsonConfigToContainerConfig(config, local))
```

Comment:
why not merge with the remote? (atm named '_')

Reply:
we talked about this in person, the scope is bigger